applications/fishing-map/src/features/workspace/workspace.mock.ts

```diff
@@ -686,6 +686,20 @@ const workspace: Workspace = {
         },
       ],
     },
+    {
+      dataviewId: 2,
+      config: {
+        color: '#FFAAAD',
+        colorRamp: 'salmon',
+      },
+      datasetsConfig: [
+        {
+          datasetId: 'carriers-tracks:v20200507',
+          params: [{ id: 'vesselId', value: '0017a7b68-87c4-5af4-c46e-5bebee3ddceb' }],
+          endpoint: 'carriers-tracks',
+        },
+      ],
+    },
```

Comment:
```suggestion
    },
    {
      dataviewId: 10
    },
     {
      dataviewId: 11
    }
```

Add background and basemap dataviews instances, to avoid the end result being a weird mix of the `dataviews` and `dataviewsConfig`.
Simply put `dataviewsConfig`, is a flat list of everything in the workspace, without having to rely on any logic.